.env.sample

```diff
@@ -1,6 +1,6 @@
 JWT_SECRET_KEY="secret"
 NODE_ENV="development"
 ELASTIC_URL="http://localhost:9200"
-ELASTIC_INDEX="marketplace"
+ELASTIC_PREFIX="marketplace"
```

Comment:
probably the name should be a bit more explicit. Like:
`ELASTIC_INDEX_NAME_PREFIX`